src/pages/PieIndexLanding.svelte

```diff
@@ -45,6 +46,17 @@
   }
 
   $: token = params.address;
+  const isBakingPie = (address) => {
```

Comment:
this is your friend https://lodash.com/docs/4.17.15#find

Reply:
@Alexintosh done.